src/dc_imex.rs

```diff
@@ -597,16 +597,19 @@ unsafe fn import_backup(context: &Context, backup_to_import: *const libc::c_char
         0,
         "Import \"{}\" to \"{}\".",
         as_str(backup_to_import),
-        as_str(context.get_dbfile()),
+        context
+            .get_dbfile()
+            .as_ref()
+            .map_or("<<None>>", |p| p.to_str().unwrap())
```

Comment:
why double `<<`?

Reply:
To make it look different from for real value.